### ISSUE TITLE
Audio can't be produced

### ISSUE THREAD

**Issue by drmjoyce** [auphonic/auphonic-mobile#6]
I recorded a training using auphonics - now i get a msg saying that the production cant be done.and msg said to report bug that needs fixed.

**Comment by drmjoyce** [comment]
I get an error message, yet the recording seems to be fine on my iphone. Just can't upload and produce it! What can i do to get this recording #12 in my list off my iphone and onto the auphonic site? Thank you

**Comment by gholzmann** [comment]
Do you see this recording on https://auphonic.com/engine/ ?
If not, you can copy this recording to for desktop computer using iTunes ...

**Comment by drmjoyce** [comment]
Hi Georg:

How are you today? 

Here's the message I left on FB and your response - and I do understand the spam stuff! It can be a bear at times…I deal with it all the time with my FB pages. 

"I recorded a presentation last night. You can hear it on my iphone, but when I hit Upload and create a production, I get a message telling me that there was an error uploading my recording. Then I get a message telling me to report a bug, which I did. But no one is getting back to me. And I have google for hours trying to find a solution or to figure out the problem. I've have come up with nothing! can anyone help me with this?
Like · · 15 hours ago

auphonic Sorry for the late answer, I don't see these messages here on facebook very good ...
However, can you send me the link to your production per email? I am sure I can help !
So the last link for my (#12) recording is: https://auphonic.com/engine/status/Rrfc429JaEiaNpNxXZkvqY 

I have tried many times to upload - but keep getting the message that it is an incomplete form. I have never had that happen before. The page where all of my recordings are is at: https://auphonic.com/engine/ (which is the same address you sent me (below). Do you mean that this incomplete form #12 recording can be saved to iTunes? 

Thanks so much for responding. 

Marilyn

---

Wishing You an Inspired Day Healthfully Ignited with E.N.E.R.G.Y.™
Dr. Marilyn Joyce, RD, The Vitality Doctor™
Your Compassionate Cancer Concierge…At Your Service! 
805-242-3464 - 888-305-1495 (24/7 Live Receptionist)

**Comment by gholzmann** [comment]
Yes, you should be able to see your recorded files in iTunes and transfer it to your computer!

**Comment by drmjoyce** [comment]
Are they filed on iTunes somewhere? I'm confused...

**Comment by gholzmann** [comment]
Sorry, I don't have an iphone here.
But you can just transfer the files as you transfer any file form your iphone to your computer (iTunes file sharing) - look in Apps and then Auphonic.

Here is a description by apple:
http://support.apple.com/kb/ht4094
